src/openapi/lab.yml

```diff
@@ -175,6 +223,7 @@ model:
       name:
         type: string
         pattern: '^\w+$'
+        maxLength: 32
```

Comment:
@brhatiga 
hyphens are allowed in FQDN, aren't they ?
do you think it's better to excluding them anyway in the regex from your experience with QL ?

Reply:
@daniellutz yes hyphens are allowed in FQDN.  There are other places where the clustername is also used such as part of the "stack" name in OSP (which also allows hyphens).

We may be able to add hyphens but we'd just need to test to make sure that I'm not forgetting some other area the clustername is used where the hyphen could cause a problem.

Reply:
I'll change the pattern and length to be sure it won't break anything.

Is there some reason for 20 char limit? DNS allows 63.

Reply:
I don't recall the reasoning for the length limit...I think we were just being conservative.  We figured folks would have plenty of room to differentiate their clusters with 20 chars and it would mean less typing for all.  It would also mean less potential issues in other places such as the length of the OSP stack name (not sure what the limit is but we add additional chars to those names) as well as display of fqdns on the webUI.